### PR TITLE
Wooc 632

### PR DESCRIPTION
**Pull request by issam-payplug**

### CODE REVIEW

File: src/Admin/Ajax.php [hunk 1]
```diff
@@ -149,6 +143,7 @@ public function handle_refresh_keys() {
 			wp_send_json_error(
```
Comment (PPmmesquita) [inline src/Admin/Ajax.php:143]: can you keep the format for the other modal msg, we'll need to standardize this

File: src/Admin/Ajax.php [hunk 2]
```diff
@@ -149,6 +143,7 @@ public function handle_refresh_keys() {
...
 				array(
 					'message' => __( 'Your account does not support LIVE mode at the moment, it must be validated first. If your account has already been validated, please log out and log in again.', 'payplug' ),
+					'still_inactive' => true
```
Comment (PPmmesquita) [inline src/Admin/Ajax.php:146]: do you need this?

**Review by PPmmesquita — CHANGES_REQUESTED** [review]

File: src/Admin/Ajax.php (hunk 2 above)
Comment (issam-payplug) [reply src/Admin/Ajax.php:146]: Yes this one come after the user try to check if his account is still inactive or it is active, if it still inactive an error message wil show in the vue side (just under the header block), else the user will be able to use live mode.

File: src/Admin/Ajax.php (hunk 1 above)
Comment (issam-payplug) [reply src/Admin/Ajax.php:143]: This is not for athe modal anymore, but for the error TextAlert

File: src/Admin/Vue.php [hunk 3]
```diff
@@ -109,7 +108,27 @@ public function payplug_section_logged() {
 					"value"   => 1, //test
 					"checked" => !PayplugWoocommerceHelper::check_mode()
 				],
-			]
+			],
+			"modal"		   => [
```
[the remark this answers is not in the document]
Comment (issam-payplug) [reply src/Admin/Vue.php]: Done

**Pull request merged**